pkg/utils/certificateUtils.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright The Ratify Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"crypto/x509"
+	"os"
+
+	"path/filepath"
+
+	"github.com/notaryproject/notation-go-lib/crypto/cryptoutil"
+)
+
+func GetCertificatesFromPath(path string) ([]*x509.Certificate, error) {
+
+	var certs []*x509.Certificate
+
+	err := filepath.Walk(path, func(file string, info os.FileInfo, err error) error {
```

Comment:
Do we need to worry about the efficiency of this walking as this is recursive and loads everything in the memory?

Reply:
Since this path is user configured we might have some time to improve this over time. If need we should open an item to track this enhancement.

Reply:
tracking item logged, https://github.com/deislabs/ratify/issues/147